This patch was developed in-house to get 'gmake test' to compile.  We have
not yet determined whether or not to pass it upstream.

--- ImageMagick-6.8.3/Magick++/demo/button.cpp.orig	2010-01-05 06:04:01.000000000 -0800
+++ ImageMagick-6.8.3/Magick++/demo/button.cpp	2015-02-11 07:03:58.267285998 -0800
@@ -90,7 +90,7 @@
     // button.display();
 
   }
-  catch( exception &error_ )
+  catch( Exception &error_ )
     {
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
--- ImageMagick-6.8.3/Magick++/demo/demo.cpp.orig	2010-05-29 18:16:06.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/demo/demo.cpp	2015-02-11 07:05:43.515927192 -0800
@@ -510,7 +510,7 @@
     //    montage_image.display();
 
   }
-  catch( exception &error_ )
+  catch( Exception &error_ )
     {
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
--- ImageMagick-6.8.3/Magick++/demo/flip.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/demo/flip.cpp	2015-02-11 07:06:56.484455695 -0800
@@ -50,7 +50,7 @@
     writeImages( morphed.begin(), morphed.end(), "flip_out.miff" );
 
   }
-  catch( exception &error_ )
+  catch( Exception &error_ )
     {
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
--- ImageMagick-6.8.3/Magick++/demo/gravity.cpp.orig	2010-05-27 13:51:26.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/demo/gravity.cpp	2015-02-11 07:08:01.108746080 -0800
@@ -73,7 +73,7 @@
     // system( "animate gravity_out.miff" );
 
   }
-  catch( exception &error_ )
+  catch( Exception &error_ )
     {
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
--- ImageMagick-6.9.4/Magick++/demo/piddle.cpp.~1~	2016-05-09 10:28:57.000000000 -0700
+++ ImageMagick-6.9.4/Magick++/demo/piddle.cpp	2016-05-10 09:08:10.587311299 -0700
@@ -170,7 +170,7 @@
     //     image.display( );
 
   }
-  catch( exception &error_ )
+  catch( Exception &error_ )
     {
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
--- ImageMagick-6.8.3/Magick++/demo/shapes.cpp.orig	2011-02-02 18:08:56.000000000 -0800
+++ ImageMagick-6.8.3/Magick++/demo/shapes.cpp	2015-02-11 07:10:26.414505806 -0800
@@ -109,7 +109,7 @@
     // image.display( );
 
   }
-  catch( exception &error_ )
+  catch( Exception &error_ )
     {
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
--- ImageMagick-6.8.3/Magick++/demo/zoom.cpp.orig	2010-05-29 18:16:06.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/demo/zoom.cpp	2015-02-11 07:12:38.302672267 -0800
@@ -63,7 +63,7 @@
             try {
               density=Geometry(argv[argv_index]);
             }
-            catch( exception &/* error_ */)
+            catch( Exception &/* error_ */)
               {
                 ParseError(argv_index,argv);
               }
@@ -119,7 +119,7 @@
             try {
               geometry=Geometry(argv[argv_index]);
             }
-            catch( exception &/* error_ */)
+            catch( Exception &/* error_ */)
               {
                 ParseError(argv_index,argv);
               }
@@ -132,7 +132,7 @@
             try {
               resample=Geometry(argv[argv_index]);
             }
-            catch( exception &/* error_ */)
+            catch( Exception &/* error_ */)
               {
                 ParseError(argv_index,argv);
               }
@@ -180,7 +180,7 @@
         }
       image.write(output_file);
     }
-    catch( exception &error_ )
+    catch( Exception &error_ )
       {
         cout << "Caught exception: " << error_.what() << endl;
         return 1;
--- ImageMagick-6.8.3/Magick++/tests/appendImages.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/appendImages.cpp	2015-02-11 07:16:52.216679885 -0800
@@ -76,11 +76,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/attributes.cpp.orig	2012-06-18 10:27:31.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/attributes.cpp	2015-02-11 07:19:39.714613173 -0800
@@ -1501,11 +1501,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
   
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/averageImages.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/averageImages.cpp	2015-02-11 07:20:22.914703162 -0800
@@ -58,11 +58,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/coalesceImages.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/coalesceImages.cpp	2015-02-11 07:21:36.319285353 -0800
@@ -45,11 +45,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/coderInfo.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/coderInfo.cpp	2015-02-11 07:22:07.803975862 -0800
@@ -118,11 +118,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
   
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/color.cpp.orig	2010-09-19 08:13:42.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/color.cpp	2015-02-11 07:25:37.141826377 -0800
@@ -138,11 +138,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
   
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/colorHistogram.cpp.orig	2010-05-27 13:51:26.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/colorHistogram.cpp	2015-02-11 07:26:23.437002967 -0800
@@ -84,11 +84,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/exceptions.cpp.orig	2011-12-31 09:22:05.000000000 -0800
+++ ImageMagick-6.8.3/Magick++/tests/exceptions.cpp	2015-02-11 07:26:55.141360186 -0800
@@ -82,12 +82,6 @@
         cout.flush();
         return 1;
       }
-    catch( exception &error_ )
-      {
-        cout << "Bogus catch: Caught exception: " << error_.what() << endl;
-        cout.flush();
-        return 1;
-      }
   
     if ( failures )
       {
--- ImageMagick-6.8.3/Magick++/tests/montageImages.cpp.orig	2011-02-02 17:31:01.000000000 -0800
+++ ImageMagick-6.8.3/Magick++/tests/montageImages.cpp	2015-02-11 07:29:32.433719660 -0800
@@ -123,11 +123,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/morphImages.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/morphImages.cpp	2015-02-11 07:30:08.668795257 -0800
@@ -54,11 +54,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.8.3/Magick++/tests/readWriteBlob.cpp.orig	2009-09-05 14:47:34.000000000 -0700
+++ ImageMagick-6.8.3/Magick++/tests/readWriteBlob.cpp	2015-02-11 07:33:35.032546179 -0800
@@ -237,11 +237,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch( exception &error_ )
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if ( failures )
     {
--- ImageMagick-6.9.4/Magick++/tests/readWriteImages.cpp.~1~	2016-05-10 09:05:23.297878177 -0700
+++ ImageMagick-6.9.4/Magick++/tests/readWriteImages.cpp	2016-05-10 09:06:14.986053932 -0700
@@ -162,11 +162,6 @@
       cout << "Caught exception: " << error_.what() << endl;
       return 1;
     }
-  catch(exception &error_)
-    {
-      cout << "Caught exception: " << error_.what() << endl;
-      return 1;
-    }
 
   if (failures)
     {
